cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -261,9 +264,16 @@ def parse_harvard_opinions(options: OptionsType) -> None:
         if not options["court_id"]:
             # Sometimes the court string doesn't match just one court
             # This is used to alleviate certain circumstances.
-            found_court = find_court_ids_by_name(
-                data["court"]["name"], bankruptcy=False
+            found_court = find_court(
+                data["court"]["name"],
+                bankruptcy=False,
             )
+            if len(found_court) > 1 and options["location"]:
```

Comment:
Why run find_court twice in this case? Shouldn't you check for the location parameter and pass it through if it's there? I bet you could probably just assign it a good default value if it's not set, and just call it the same no matter what?

Reply:
This is probably true.

Reply:
resolving this.